src/models/data.h

```diff
@@ -233,6 +233,11 @@ struct BottomUp : SymbolTree<BottomUp>
     Cost cost;
 };
 
+inline bool operator<(const BottomUp& lhs, const BottomUp& rhs)
```

Comment:
this is not generic, please pass this as a lambda/comparator to wherever you need it in the tests